pyodide_build/config.py

```diff
@@ -240,6 +243,8 @@ def _get_make_environment_vars(self) -> Mapping[str, str]:
     "build_dependency_index_url": "https://pypi.anaconda.org/pyodide/simple",
     "default_cross_build_env_url": "",
     "xbuildenv_path": "",
+    # A list of PEP508 build-time requirements to be ignored when building a wheel
+    "ignored_build_requirements": " ".join(BASE_IGNORED_REQUIREMENTS),
```

Comment:
(no action requird for this PR)

It would be great if we could support List types here. The question is how to support the list type in an env variable.

Reply:
Yes, I agree – I faced this problem when implementing it. I feel there is no neat option to do it in this case, as `cibuildwheel` also largely suffers from the same limitation.